tests/test_femtonics.py

```diff
@@ -0,0 +1,149 @@
+import numpy as np
+from numpy import dtype
+from numpy.testing import assert_array_equal
+from datetime import datetime, timezone
+
+from roiextractors.extractors.femtonicsimagingextractor import FemtonicsImagingExtractor
+from .setup_paths import OPHYS_DATA_PATH
+
+
+def test_femtonicsimagingextractor_p29_mesc():
+    """
+    Test with Femtonics p29.mesc file.
+
+    File: p29.mesc
+    Metadata (note: metadata vs actual data mismatch):
+    - Source: Femtonics .mesc file (MESc 3.3)
+    - Metadata claims: 56250 samples (frames)
```

Comment:
Thanks for doing this docstrings. This mismatch that you detected here is my fault. I stubbed the file (i.e. made a smaller version of it) and I changed the frames but did not change the metadata. Thanks for noticing.